styles.css

```diff
@@ -0,0 +1,219 @@
+* {
+  box-sizing: border-box;
+}
+
+body {
+  font-size: 62.5%;
+  margin: 0;
+  font-family: Verdana, Geneva, Tahoma, sans-serif;
+}
+
+/* header */
+
+header {
+  width: 100%;
+}
+
+header .container {
+  margin: 30px auto;
+  width: 40%;
+  display: flex;
+  justify-content: center;
+  justify-content: space-between;
+  /* position: absolute; */
```

Comment:
if you dont need this, please remove it

Reply:
using this as a place-holder, will be using position: absolute to have this section hover over section below